debian/ceph-mgr-modules-core.install

```diff
@@ -15,8 +15,6 @@ usr/share/ceph/mgr/pg_autoscaler
 usr/share/ceph/mgr/progress
 usr/share/ceph/mgr/prometheus
 usr/share/ceph/mgr/rbd_support
-usr/share/ceph/mgr/rgw
```

Comment:
Why is this line being removed?

Reply:
No clue... Probably wrong conflict solving, as it's near the modified line below.